101-sqlmi-new-vnet/azuredeploy.json

```diff
@@ -0,0 +1,170 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.1",
+    "parameters": {
+        "managedInstanceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Enter managed instance name."
+            }
+        },
+        "administratorLogin": {
+            "type": "string",
+            "defaultValue": "cloudSA",
+            "metadata": {
+                "description": "Enter user name."
+            }
+        },
+        "administratorLoginPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Enter password."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "",
```

Comment:
use
[resourceGroup().location]
for the defaultValue

Reply:
done